pywarp10/pywarp10.py

```diff
@@ -55,13 +59,15 @@ def __init__(
         self,
         host: Optional[str] = None,
         port: Optional[int] = None,
-        connection: Literal["py4j", "html"] = "py4j",
+        connection: Literal["py4j", "http"] = "py4j",
         **kwargs,
     ) -> None:
         """Inits Warpscript with default host and port"""
         if connection not in ["py4j", "http"]:
             raise ValueError("connection must be either py4j or http.")
         self.host = host or os.getenv("WARP10_HOST", "127.0.0.1")
+        if connection == "http":
+            self.host = f"{self.host}/api/v0/exec"
```

Comment:
what about the write url: `http://127.0.0.1:8080/api/v0/update` ?

Reply:
This is not intended for now with this package.